scapy/contrib/wpa_eapol.py

```diff
@@ -41,6 +43,6 @@ def answers(self, other):
         if isinstance(other,WPA_key):
                return 1
         return 0
-             
+
```

Comment:
Please remove this useless change.

Reply:
ok! I understand the policy of keeping the history clean.
But usually white spaces should be removed.